src/os/ui/arc/iarcdescriptor.js

```diff
@@ -0,0 +1,52 @@
+goog.provide('os.ui.arc.IARCDescriptor');
+
+goog.require('os.data.IDataDescriptor');
+goog.require('os.data.IServerDescriptor');
+goog.require('os.filter.IFilterable');
+goog.require('os.ui.ogc.IFeatureTypeDescriptor');
+
+/**
+ * Interface for ARC data descriptors
+ * @interface
+ * @extends {os.data.IDataDescriptor}
+ * @extends {os.data.IServerDescriptor}
+ * @extends {os.filter.IFilterable}
+ * @extends {os.ui.ogc.IFeatureTypeDescriptor}
+ */
+os.ui.arc.IARCDescriptor = function() {};
+
+/**
+ * @type {string}
+ * @const
+ */
+os.ui.arc.IARCDescriptor.ID = 'os.ui.arc.IARCDescriptor';
+
+/**
+ * @return {?string}
+ */
+os.ui.arc.IARCDescriptor.prototype.getUrlKey;
```

Comment:
Can we please not add this to the API? See https://github.com/ngageoint/opensphere/issues/545

Reply:
I was planning on merging that after this, but it looks like Kevin already merged it, so you're correct. This should not be added.

Reply:
Sorry I should have checked first. I wanted to avoid adding this, then removing it again later.

Reply:
I have removed it in the most recent push. I think it got added to the merge commit